File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/uncomment/direct/Java/Go/s466963472_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func solve(d int) int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		ans += x * x * d
	}
	return ans
}

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Split(bufio.ScanWords)
	for scanner.Scan() {
		token := scanner.Text()
		d, err := strconv.Atoi(token)
		if err != nil {
			continue
		}
		fmt.Println(solve(d))
	}
	if err := scanner.Err(); err != nil {
		fmt.Fprintln(os.Stderr, "reading standard input:", err)
	}
}

// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/uncomment/direct/Java/C/s466963472_commented.c
#include <stdio.h>

int d;

int solve() {
    int ans = 0;
    for (int i = 1; i * d < 600; i++) {
        int x = i * d;
        ans += x * x * d;
    }
    return ans;
}

int main() {
    while (scanf("%d", &d) == 1) {
        printf("%d\n", solve());
    }
    return 0;
}
// End of Code
